lib/cloudex.ex

```diff
@@ -9,6 +9,7 @@ defmodule Cloudex do
 
       start %{api_key: "key", secret: "s3cr3t", cloud_name: "heaven"}
   """
+  @default_recv_timeout 2000
```

Comment:
default recv_timeout for httpoison is 5000, so we shouldn't be setting our own, we should default it to 5000